### PR TITLE
Implement monolog

### PR DESCRIPTION
**Pull request by Tjoosten**

### CODE REVIEW

File: api/APICall.php [hunk 1]
```diff
@@ -119,40 +121,33 @@ protected function writeLog($ua, $from, $to, $err, $ip) {
...
-
-             mysql_pconnect($_ENV['apiHost'], $_ENV['apiUser'], $_ENV['apiPassword']);
-             mysql_select_db($_ENV['apiDatabase']);
-	  }
-	  catch (Exception $e) {
-	       throw new Exception("Error connecting to the database.", 3);
-	  }
-     }
+	public static function connectToDB(){
+		try {
```
Comment (pietercolpaert) [inline api/APICall.php]: These are tabs, not 4 spaces indent?

Comment (Tjoosten) [inline api/APICall.php]: the code isn't reformatted. I will do that in another pr.

**Comment by pietercolpaert** [comment]
Couple of questions:
 * did you test this code?
 * can't we remove all the DB connection and .env dependencies now? Thanks to Monolog we are not using the DB any longer
 * What's the output of the Monolog? Is it well structured? I remember the DB was a nightmare ;)

**Comment by Tjoosten** [comment]
- Yes the code is tested. 
- No because there are some stations that are getted from a database. 
- Output: 

```
[2015-08-17 11:58:58] irail_api.INFO:  ["Mon, 17 Aug 2015 11:58:58","Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_4) AppleWebKit/600.7.12 (KHTML, like Gecko) Version/8.0.7 Safari/600.7.12","EMPTY","EMPTY","Error in vehicleinformation id not set. Please review your request and add the right parameters","::1","0"] []

```

**Comment by pietercolpaert** [comment]
1. Great!
2. I've removed all code that retrieves stations from the database for NMBS. I'd like to delete support for NS, MIVB and others: I'm not going to maintain this interface anymore for these transit modes. Reusers should first start here if they want intermodal routes: http://gtfs.irail.be
3. Nice start! I'll pull this in on a feature-branch and work further on your work to create better overall logging!

**Comment by Tjoosten** [comment]
alright! what's next what i can work on

File: composer.json [hunk 2]
```diff
@@ -10,7 +10,8 @@
   ],
   "require": {
     "vlucas/phpdotenv": "^2.0",
-    "irail/stations": "dev-master"
+    "irail/stations": "dev-master",
+    "monolog/monolog": "^1.16@dev"
```
Comment (pietercolpaert) [inline composer.json]: Any particular reason why you chose this version?

Comment (Tjoosten) [inline composer.json]: Not really

**Comment by Tjoosten** [comment]
This PR need to be done against the development branch. Not the master

**Comment by pietercolpaert** [comment]
Implemented myself

**Pull request closed without merging**